### DISCUSSION TITLE
Github has started work very slowly

### DISCUSSION THREAD

**Discussion by JeanPaulLucien** [community/community#57797] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

About 30 minutes before Github has started work very slow. My PC works like it's high loaded. But all actions on Github webpages spend much time, more 15 seconds.

**Comment by JeanPaulLucien** [comment]
Hmm... While I was writing situation has change, now it's better.

**Comment by liveFreeOrCode** [comment]
I am still experiencing this, on all browsers I have: Chrome, Firefox and Edge.  On Windows 10.  It's almost impossible to work with.  Memory and power consumption are off the charts

**Comment by sainak** [comment]
I've also been experiencing these lag spikes from the past couple of weeks.
The links take much longer to load in case of client side router and sometimes clicks don't work at all.
Doing a hard refresh fixes the issue temporarily.

**Comment by catalpaaa** [comment]
same here
